src/utils.js

```diff
@@ -3,7 +3,11 @@ var stringify = require('../vendor/json-stringify-safe/stringify');
 var _window =
   typeof window !== 'undefined'
     ? window
-    : typeof global !== 'undefined' ? global : typeof self !== 'undefined' ? self : {};
+    : typeof global !== 'undefined'
```

Comment:
The lint script runs prettier automatically, and it did this to the file.

I verified in my yarn.lock that I'm on the correct prettier version; I can skip the lint step if you'd prefer and return this to it's normal formatting if y'all are getting different behavior, just let me know.